integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/table/CarbonDescribeFormattedCommand.scala

```diff
@@ -92,7 +92,9 @@ private[sql] case class CarbonDescribeFormattedCommand(
         Strings.formatSize(
           tblProps.getOrElse(CarbonCommonConstants.CARBON_LOAD_MIN_SIZE_INMB,
             CarbonCommonConstants.CARBON_LOAD_MIN_SIZE_INMB_DEFAULT).toFloat), ""),
-
+      ("Carbon Column Compressor ", tblProps
```

Comment:
Same compressor is used for column and row data files. So, please change it to  "Data File Compressor"

Reply:
Changed to "Data File Compressor"
As this is in table properties we are displaying default value.It's same for other properties also,

Reply:
OK